hanadb_exporter/db_manager.py

```diff
@@ -0,0 +1,95 @@
+"""
+SAP HANA database manager
+
+:author: xarbulu
+:organization: SUSE Linux GmbH
+:contact: xarbulu@suse.de
+
+:since: 2019-10-24
+"""
+
+import logging
+import time
+
+from shaptools import hdb_connector
+from hanadb_exporter import utils
+
+RECONNECTION_INTERVAL = 15
+
+
+class DatabaseManager(object):
```

Comment:
about naming. I was thinking if we could specialize this more with

`DbConnectionManager` class